packages/atomic/src/components.d.ts

```diff
@@ -1462,9 +1462,13 @@ export namespace Components {
          */
         "clearFilters": boolean;
         /**
-          * Whether to prevent the user from triggering a search from the component. Perfect for use cases where you need to disable the search conditionally, like when the input is empty.
```

Comment:
@jpmarceau @srosenberg-coveo Please review the doc changes. 
* Starting the sentence with `Whether` (conjunction) feels weird. Although it is not invalid grammatically I think it reads weird. 
* But I do see that this pattern is used in many places in this repo. So inclined to follow the same for consistency. Let me know your thoughts ..

Reply:
Hello @mrrajamanickam-coveo 👋 
Yeah, `Whether...` is a pattern we use tyically in reference docs, not just in ui-kit. We use it specifically for booleans. If we start with a verb (e.g., "Prevent ..."), it can seem, at first glance, like we're documenting a function rather than a boolean.
Writing "Whether..." for booleans seems common in the industry, so I don't find it weird 🙂 For example, [Algolia](https://www.algolia.com/doc/api-reference/api-parameters):
![Screen Shot 2023-04-17 at 12 07 36 PM](https://user-images.githubusercontent.com/39384459/232545000-9c545b69-9895-4604-a2a5-00d6a4a287a8.png)

Do you think that makes sense?

Reply:
Thanks for the explanation @jpmarceau 
Will revert to using `Whether ..`